dragonfly/building.py

```diff
@@ -0,0 +1,241 @@
+# coding: utf-8
+"""Dragonfly Building."""
+from ._base import _Base
+from .properties import BuildingProperties
+from .story import Story
+
+from honeybee.model import Model
+
+from ladybug_geometry.geometry3d.vector import Vector3D
+
+
+class Building(_Base):
+    """A complete Building defined by Stories.
+
+    Properties:
+        * name
+        * display_name
+        * unique_stories
+        * multipliers
+        * all_stories
+        * unique_room_polygons
+        * all_room_polygons
+        * footprint_geometry
+        * context_geometry
+        * height
+        * volume
+        * floor_area
+        * exterior_wall_area
+        * exterior_aperture_area
+    """
+    __slots__ = ('_unique_stories', '_multipliers')
+
+    def __init__(self, name, unique_stories, multipliers=None):
```

Comment:
I feel you can include the `multiplier` as an attribute for `Story` class so it can be set in Stroy itself before adding it to building.

Reply:
Or you might create a new class that subclasses from Story. Let's say `MultipliedStroy(Stroy)`?

Reply:
Can you explain the advantage of that over the current implementation?

Reply:
I think it is a cleaner implementation. Concept of multiplier can be directly applied to a story. Generally speaking whenever I see inputs like this that needs matching between two inputs I think of merging them into the reference object itself. Things can go wrong with data-matching and avoiding them makes the code less prone to errors.

Reply:
Ok. I see what you mean. And I guess we don't have many cases where we want to have the Story divorced from its multiplier. Let me implement multipliers correctly on honeybee-core first (I agree that we definitely want them to be assigned to the `Room` there). Then, I'll revisit multipliers here and that will give me some time to mull over how everything could work here.